functions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+function getDB(): PDO {
```

Comment:
you might put the { on a separate line (Mike/Neal/Lucia might have a problem with that, but I am not sure)

Reply:
Further to Alicja's comment, yes, the `{` goes on its own line when it's the opening one of a function.
And the following lines should be indented.